packages/grid/x-data-grid/src/tests/columnsGrouping.DataGrid.test.tsx

```diff
@@ -293,6 +292,7 @@ describe('<DataGrid /> - Column grouping', () => {
   describe.skip('error messages', () => {
     function TestWithError(props: TestDataGridProps) {
       return (
+        // @ts-ignore TODO: check this
         <ErrorBoundary logger={console}>
```

Comment:
```suggestion
        <ErrorBoundary>
```

Looks like ErrorBoundary never supported `logger` prop.
This test is skipped anyway 🤷🏻‍♂️